YaqeenDAL/Model/Model.cs

```diff
@@ -210,4 +210,29 @@ public class ResourceLocalization
         [Column(TypeName = "jsonb")]
         public Dictionary<string, string> Translation { get; set; }
     }
-}
\ No newline at end of file
+        public class Country
+    {
+        [Key]
```

Comment:
Please add the Database generated annotation... It should be in other tables to copy from